src/Avalonia.Visuals/Media/FontFamily.cs

```diff
@@ -0,0 +1,119 @@
+﻿// Copyright (c) The Avalonia Project. All rights reserved.
+// Licensed under the MIT license. See licence.md file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Avalonia.Media.Fonts;
+
+namespace Avalonia.Media
+{
+    public class FontFamily
+    {
+        internal static FontFamily Default = new FontFamily("Courier New");
```

Comment:
Can we use the same font that is currently default? Some apps may be relying on this being consistent.

This should be a font or series of fonts available cross platform, perhaps we can see what WPF default it, and have font fallbacks to similar fonts for Linux, mac and ios / android.

Reply:
I just used what already was the default just defined it as a static property. This should be used everywhere in our code base when the default is needed.